bitbots_buttons/src/bitbots_buttons/button_node.py

```diff
@@ -43,7 +43,8 @@ def __init__(self):
             try:
                 rospy.wait_for_service("manual_penalize", 0.5)
             except rospy.exceptions.ROSException as exc:
-                rospy.logerr("service 'manual_penalize' not available. Please start HCM")
+                pass
+                #rospy.logerr("service 'manual_penalize' not available. Please start HCM")
```

Comment:
@SammyRamone

Reply:
Yes should actually be ignored. Button node is now lauched with ros control and not with HCM. So it can not expect that the HCM is running. I did remove the old commented file and added an explaination